### DISCUSSION TITLE
geomap legend information: threshold values and/or values mapping

### DISCUSSION THREAD

**Discussion by chaos-user** [grafana/grafana#52600] [Dashboards, Panels & Visualizations]
Add possibility to choose at legend data information based on threshold values and/or values mapping.
This allow user to understand and visualize is data much more accurately

![geomap legend](https://user-images.githubusercontent.com/75700201/180201491-389bf1eb-c08e-43db-935c-bd6a65dff2df.jpg)
.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
